From 4a6692d25e6ea4032daf942d319229d750075299 Mon Sep 17 00:00:00 2001
From: b07117 <b07117@freescale.com>
Date: Thu, 7 Apr 2011 11:35:38 +0800
Subject: [PATCH 133/498] ENGR00000000, Fixed hardware/mx5x building error for
 imx508 board

Changed linked library to libc2d_z160 for imx508 board

Signed-off-by: xianzhong <b07117@freescale.com>
---
 libgralloc/Android.mk | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/libgralloc/Android.mk b/libgralloc/Android.mk
index 290dfc3..0f1fb2d 100755
--- a/libgralloc/Android.mk
+++ b/libgralloc/Android.mk
@@ -20,7 +20,12 @@ LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 LOCAL_PRELINK_MODULE := false
 LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
-LOCAL_SHARED_LIBRARIES := liblog libcutils libGLESv1_CM libipu libc2d_z430
+LOCAL_SHARED_LIBRARIES := liblog libcutils libGLESv1_CM libipu
+ifeq ($(BOARD_SOC_TYPE),IMX50)
+LOCAL_SHARED_LIBRARIES += libc2d_z160
+else
+LOCAL_SHARED_LIBRARIES += libc2d_z430
+endif
 LOCAL_C_INCLUDES += external/linux-lib/ipu
 LOCAL_C_INCLUDES += hardware/mx5x/libcopybit
 
-- 
1.8.0

